### PR TITLE
[docs-only]update test docs related to email notification

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
### Description
Update the documentation on running the oCIS for email notification. Added the extra envs.

### CODE REVIEW

File: docs/ocis/development/testing.md [hunk 1]
```diff
@@ -364,6 +364,10 @@ NOTIFICATIONS_SMTP_HOST=localhost \
 NOTIFICATIONS_SMTP_PORT=2500 \
 NOTIFICATIONS_SMTP_INSECURE=true \
 NOTIFICATIONS_SMTP_SENDER="owncloud <noreply@example.com>" \
+EMAIL_HOST=localhost \
+EMAIL_PORT=9000 \
```
Comment (saw-jan) [inline docs/ocis/development/testing.md:368]: These envs are only used by test code

File: docs/ocis/development/testing.md [hunk 2]
```diff
@@ -364,6 +364,10 @@ NOTIFICATIONS_SMTP_HOST=localhost \
...
+NATS_NATS_HOST=0.0.0.0 \
+NATS_NATS_PORT=9233 \
```
Comment (saw-jan) [inline docs/ocis/development/testing.md:370]: hmm :thinking: , I don't think we need this.

Comment (saw-jan) [reply docs/ocis/development/testing.md:370]: tests are working with the current docs.

**Comment by PrajwolAmatya** [comment]
It didn't work before and worked when adding these envs. But with the current docs it is working fine now. So, closing this issue for now.

**Pull request closed without merging**